scrapy/core/scheduler.py

```diff
@@ -13,9 +12,11 @@
 
 class Scheduler(object):
 
-    def __init__(self, dupefilter, jobdir=None, dqclass=None, mqclass=None, logunser=False, stats=None):
+    def __init__(self, dupefilter, jobdir=None, logunser=False, stats=None,
+                 pqclass=None, dqclass=None, mqclass=None):
```

Comment:
+1 to use kwargs in from_crawler; it won't help with backwards compatibility (because users may still use *args), but it'll make code cleaner and less likely to break.